package.json

```diff
@@ -6,15 +6,23 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "lint": "eslint . --ext .ts"
   },
   "devDependencies": {
+    "@eslint/js": "^9.28.0",
+    "@typescript-eslint/eslint-plugin": "^8.34.0",
+    "@typescript-eslint/parser": "^8.34.0",
     "@webgpu/types": "^0.1.61",
+    "eslint": "^9.28.0",
+    "globals": "^16.2.0",
```

Comment:
small thing but I don't think you need this package? I don't see it being used anywhere